src/Security/LogoutHandler.php

```diff
@@ -79,6 +79,8 @@ private function triggerPostLogoutHook(User $user): void
 
         @trigger_error('Using the postLogout hook has been deprecated and will no longer work in Contao 5.0. Use the contao.post_logout event instead.', E_USER_DEPRECATED);
 
+        $GLOBALS['TL_USERNAME'] = $user->getUsername();
```

Comment:
I don't think so Tim.

Reply:
This was previously available (https://github.com/contao/core-bundle/blob/4.4/src/Resources/contao/library/Contao/User.php#L683), there's no reason to remove it if we want to keep BC with the hook.